.github/workflows/nodejs.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       no-lockfile: true
       npm-test-script: 'test-ci'
       # Node 16 has a SegFault from C8 when using --test
-      node-versions: '18,19'
+      node-versions: '19,20'
       # We currently have some issues on Windows that will have to wait to be fixed
```

Comment:
The typosquatting test fails in CI with version 18 so not sure why but it passes when updating the versions.